pkg/kubelet/dockershim/docker_sandbox.go

```diff
@@ -36,7 +36,7 @@ import (
 )
 
 const (
-	defaultSandboxImage = "gcr.io/google_containers/pause-amd64:3.1"
```

Comment:
can this start use `test/utils/image` instead?

Reply:
I'm little hesitant to use this one because it is completely outside of `pkg/` package.

Reply:
I agree with @mkumatag - I think we should avoid dependencies from `pkg/` -> `test/`.